rapid/client/executor.py

```diff
@@ -92,15 +94,39 @@ def verify_work_request(self):
             raise Exception("No command was set.")
 
     def _create_logger(self):
-        logger = logging.getLogger("rapid")
+        log_file_name = self._get_log_file_name()
+        if not log_file_name:
+            logger = logging.getLogger('rapid')
+            logger.setLevel(logging.INFO)
+            return logger
+
+        logger = logging.getLogger("rapid-executor")
         logger.setLevel(logging.INFO)
+
+        self._file_handler = logging.FileHandler(log_file_name)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        self._file_handler.setFormatter(formatter)
+        logger.addHandler(self._file_handler)
+        
         return logger
 
+    def _get_log_file_name(self):
+        if self.rapid_config and self.rapid_config.workspace and self.rapid_config.log_to_directory:
```

Comment:
Again, because I know little: why are we checking self.rapid_config.workspace? it doesn't seem to be used below?

Reply:
I shouldn't be....left over code!